routes/web/hardware.php

```diff
@@ -178,13 +179,17 @@ function () {
         Route::post('bulkcheckout',
             [BulkAssetsController::class, 'storeCheckout']
         )->name('hardware.bulkcheckout.store');
+
     });
 
 Route::resource('hardware', 
         AssetsController::class, 
         [
             'middleware' => ['auth'],
-            'parameters' => ['asset' => 'asset_id'
+            'parameters' => ['asset' => 'asset_id',
+                'names' => [
+                'show' => 'view',
```

Comment:
It might be obvious but what's up with the `'names' => ['show' => 'view']`?

Reply:
This was just belt and suspenders in case I missed a route to `hardware.view` somewhere (we use `hardware.show` via resource routes everywhere else.)